dev-vcs/qgit/qgit-2.9.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A Qt-based git GUI viewer"
+DESCRIPTION="With qgit you will be able to browse revisions history,\
+view patch content and changed files, graphically following different \
+development branches."
+HOMEPAGE="https://github.com/tibirna/qgit"
+COPYRIGHT="2011-2019 Cristian Tibirna"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/tibirna/qgit/archive/qgit-$portVersion.tar.gz"
+CHECKSUM_SHA256="69788efda7d434d1bc094bb414bd92c269dc7894326320634500b05d63c008e8"
+SOURCE_DIR="qgit-qgit-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	qgit$secondaryArchSuffix = $portVersion
+	cmd:qgit
+	app:QGit
+	"
+REQUIRES="
+	haiku
+	cmd:git
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	qthaikuplugins$secondaryArchSuffix
```

Comment:
Should not depend on qthaikuplugins.
```suggestion
```

Reply:
The older version of my recipe does not include `qthaikuplugins`. However, when installing the resulting `.hpkg` file, QGit says that `qthaikuplugins` are missing and opens a dialog allowing the user to install it.
Adding this line solves this problem.

Reply:
![Haikux64_clean-2020-01-21-12-52-53](https://user-images.githubusercontent.com/57174311/72778936-f4c5b480-3c4c-11ea-876d-c73501a2e37c.png)

This is what running the QGit executable while having `qthaikuplugins` uninstalled give.

Reply:
Any take on this, on bare install these plugins aren't installed?

Reply:
Every Qt application will show this popup. The dependency is not "hard". On a blank system installing QGit will install the required and recommended packages.

Reply:
Clear enough :)

Reply:
@trungnt2910 can you take care of this?